source/administration/changelog.md

```diff
@@ -4,6 +4,135 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ - The config.json file is now generated with build time using defaults in code and not in ``default.json``.
+ - Added new settings to have more control over ``BindAddress`` and ``AdvertiseAddress`` in the cluster server to allow users to configure properly in situations where the servers are communicating through another server using NAT.
+ 
+#### Enterprise Edition (EE)
+ - Added support for signing SAML requests to be able to be approved to Infosec.
+ - Added support for configuring the device to detect the IP in High Availability clusters.
+
+### Bug Fixes
+ - Fixed an issue where bulk import got stuck when importing lines were missing the "type" entry.
+ - Fixed an issue where disabling email notifications also disabled email invites.
+ - Fixed an issue where Admins were shown a warning of a user's bot being deactivated even if they already were.
+ - Fixed an issue where a bot profile image disappeared when saving bot details.
+ - Fixed an issue where plus-sign was not visible on mobile browser view for reacting with a new emoji next to existing reactions.
+ - Fixed an issue in the System Console where the UserID in User Activity Logs changed from email to UserID.
+ - Fixed an issue where user got a notification to add a bot to a channel when mentioning it.
+ - Fixed an issue where permanenently deleting a bot user didn't remove it from the bots table.
+
+### config.json
+
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+
+
+
+### Database Changes
+
+### API Changes
+ - Migrated user API endpoint from Google+ API to People API.
+ - Added promote/demote API endpoint.
+ - Added invite/remove/promote API endpoints.
+ - Added ``/search_group`` API endpoint to return the group channels whose members' usernames match the search term.
+ - Added "users who will be removed from team" API endpoint.
+ - Added "users who will be removed from channel" API endpoint.
+ - Added API to retrieve paginated list of teams, including total count.
+ - Added API to retrieve paginated list of channels, including total count.
+ - Added API to get users modified since a given time.
+ - Added "GET /posts/unread" API to support landing on the last unread post.
+ - Added JSON versions of the KVStore plugin API.
+ - Added GET POST DELETE handlers for /api/v4/bots/<BOT_USER_ID>/icon endpoint.
+
+#### Plugin API
+ - Extended PluginAPI to include ``GetBotIconImage``, ``SetBotIconImage`` and ``DeleteBotIconImage``.
+
+### Websocket Event Changes
+
+### Known Issues
+ - Buttons inside ephemeral messages are not clickable / functional on the mobile app.
+ - On a server using a subpath, the URL opens a blank page if the System Admin changes the Site URL in the System Console UI. To fix, the System Admin should restart the server.
```

Comment:
Could we just document this as expected behaviour? I don't see us ever fixing this, per se.